lib/errors.js

```diff
@@ -0,0 +1,58 @@
+'use strict';
+
+const errorTitle = (code) => `${code}_ERROR`;
+const errorMsg = (code) => `There is a ${code}_ERROR`;
+
+const getContent = (err, debug, translate) => {
+  let content = {};
```

Comment:
no, you are not changing the assignment, properties can be added/amended/deleted on the fly, it is still the same object

However if you were to do `let foo = 'bar'; foo += 'baz';` this would error as you are altering the assignment `foo = foo + 'baz'`